owncloudApp/src/main/java/com/owncloud/android/MainApp.kt

```diff
@@ -158,21 +156,12 @@ class MainApp : Application() {
         initDependencyInjection()
     }
 
-    fun startLogIfDeveloper() {
-        isDeveloper =
-            BuildConfig.DEBUG || PreferenceManager.getDefaultSharedPreferences(applicationContext)
-                .getInt(CLICK_DEV_MENU, CLICKS_DEFAULT) > CLICKS_NEEDED_TO_BE_DEVELOPER
+    fun startLogsIfEnabled() {
+        enabledLogging = PreferenceManager.getDefaultSharedPreferences(applicationContext)
+                .getBoolean(SettingsLogsFragment.PREFERENCE_ENABLE_LOGGING, false)
 
-        if (isDeveloper) {
-            val dataFolder = dataFolder
-
-            // Set folder for store logs
-            LoggingHelper.startLogging(
-                File(Environment.getExternalStorageDirectory().absolutePath + File.separator + dataFolder), dataFolder
-            )
-            Timber.d("${BuildConfig.BUILD_TYPE} start logging ${BuildConfig.VERSION_NAME} ${BuildConfig.COMMIT_SHA1}")
```

Comment:
It's not smart to remove it. 
With it you see in logfile, when the was started and it makes absolute clear which version you are using

Reply:
Sorry, I see now it's here https://github.com/owncloud/android/pull/3150/files?file-filters%5B%5D=.java&file-filters%5B%5D=.kt&file-filters%5B%5D=.xml&hide-deleted-files=true#diff-67893595fc2d9801bbc5624857ff2527b1ffc6c7a6ad30c763d054590bfd1a73R44

Please forget my comment